test/test_rpc.py

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env python3
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import numpy
```

Comment:
why do we need this import?

Reply:
it tried to see whether it can fix test error about "numpy + intel MKL", but I removed in new diff, will see whether tests passed